src/main/scala/higherkindness/rules_scala/workers/zinc/compile/ZincRunner.scala

```diff
@@ -223,7 +223,9 @@ object ZincRunner extends WorkerMain[Namespace] {
     }
 
     val setup = {
-      val incOptions = IncOptions.create()
+      val incOptions = IncOptions
+        .create()
+        .withAuxiliaryClassFiles(Array(TastyFiles.instance()))
```

Comment:
This just tells Zinc we want it to include TASTy files when it's deciding what to cache for incremental compilation, right?